src/Repositories/MovieRepository.cs

```diff
@@ -68,5 +68,9 @@ orderby m.Genre
             vo.Genres = new SelectList(await genreQuery.Distinct().ToListAsync());
             return vo;
         }
+
+        public Movie GetMovieWithComments(int id) {
+            return Context.Movies.Where(m => m.Id == id).Include(m => m.Comments).FirstOrDefault();
```

Comment:
Just as a comment, remember that when you search by id you can use Find, for example
`return Context.Movies.Find(id)`
but in this case is not possible, because you need to use the Include :P
So this is ok, its just a comment haha